src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -233,11 +234,11 @@ const TopicFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: colors.white
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constants have been updated correctly from `COLORS.white` to `colors.white`. However, please ensure that all other instances of `COLORS.*` in this file (and across the project) are also updated to `colors.*` to maintain consistency.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: colors.white
```
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->